prow/plugins/trigger/pull-request.go

```diff
@@ -225,6 +228,10 @@ func buildAllIfTrusted(c Client, trigger plugins.Trigger, pr github.PullRequestE
 	if err != nil {
 		return fmt.Errorf("could not validate PR: %s", err)
 	} else if trusted {
+		if pr.PullRequest.Draft && !github.HasLabel(labels.OkToTest, l) {
```

Comment:
I think we should remove the second condition, because even if the PR had `ok-to-test` it wouldn't get built as long as its a draft

Reply:
Sure, i wanted to keep the ok-to-test idea so when the label is there, the tests run no matter what.
But i'm okay with both.

Reply:
The `ok-to-test` label has no relevance to org members. It only changes behavior for people foreign to the org.